build_scripts/versions.yaml

```diff
@@ -35,67 +35,6 @@ postgres_versions:
   13: 13.22
 
 timescaledb:
-  2.1.0:
-    pg: [13]
-  2.1.1:
-    pg-max: 13
-  2.2.0:
-    pg-max: 13
-  2.2.1:
-    pg-max: 13
-  2.3.0:
-    pg-max: 13
-  2.3.1:
-    pg-max: 13
-  2.4.0:
-    pg-max: 13
-  2.4.1:
-    pg-max: 13
-  2.4.2:
-    pg-max: 13
-  2.5.0:
-    pg-max: 14
-  2.5.1:
-    pg-max: 14
-  2.5.2:
-    pg-max: 14
-  2.6.0:
-    pg-max: 14
-  2.6.1:
-    pg-max: 14
-  2.7.0:
-    pg-max: 14
-  2.7.1:
-    pg-max: 14
-  2.7.2:
-    pg-max: 14
-  2.8.0:
-    pg-max: 14
-  2.8.1:
-    pg-max: 14
-  2.9.0:
-  2.9.1:
-  2.9.2:
-  2.9.3:
-  2.10.0:
-  2.10.1:
-  2.10.2:
-  2.10.3:
-  2.11.0:
-  2.11.1:
-  2.11.2:
-  2.12.0:
-    pg-min: 13
-  2.12.1:
-    pg-min: 13
-  2.12.2:
-    pg-min: 13
-  2.13.0:
-    pg-min: 13
-    pg-max: 16
-  2.13.1:
-    pg-min: 13
-    pg-max: 16
```

Comment:
Are we deprecating all those versions too? If that's the case, aren't they also supported for pg15+?

Reply:
This isn't deprecating them but removing the checks, I will add 2.13 back.